pkg/api/dtos/folder.go

```diff
@@ -22,6 +22,8 @@ type Folder struct {
 	Updated       time.Time              `json:"updated"`
 	Version       int                    `json:"version"`
 	AccessControl accesscontrol.Metadata `json:"accessControl,omitempty"`
+	// only used if nested folders are enabled
+	ParentUID string `json:"parent_uid"`
```

Comment:
```suggestion
	ParentUID string `json:"parentUid"`
```

Reply:
Sorry for murdering the snake here, but I think we agreed to prefer camels over snakes yesterday